.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
Why would you use prettier... prettier is just for formatting. And xo already supports it out of the box.

Reply:
Is not a big deal man you can remove, because we are 2 contributors so I just wanted to make sure we get the same code formatting but I didn't know about xo.